android/assets/jsons/Civ V - Gods & Kings/UnitPromotions.json

```diff
@@ -227,6 +227,11 @@
 		"uniques": ["[+20]% Strength <vs cities>", "Earn [33]% of the damage done to [City] units as [Gold]"],
 		"unitTypes": ["Melee Water"]
 	},
+	{
+		"name": "Amphibious",
+		"uniques": ["Eliminates combat penalty for attacking across a coast"],
+		"unitTypes": ["Melee Water"]
```

Comment:
Wiki says this is for melee units?
https://civilization.fandom.com/wiki/Promotions_(Civ5)

Reply:
This PR adds an additional Amphibious promotion, different from the already existing one for melee units

Reply:
In that case it's even worse, two promotions with the same name won't work, the latter will override the former
We save the promotions in a String - Promotions hashmap

Reply:
Oooh, ok. Will fix